appveyor.yml

```diff
@@ -3,7 +3,6 @@ environment:
   matrix:
     - nodejs_version: "6"
     - nodejs_version: "8"
-    - nodejs_version: "10"
```

Comment:
why don't we test on `nodejs 10` ?

Reply:
Because it takes too long to run all platforms and node 10 should generally work if node 8 works

Reply:
Okay, that makes sense. Hopefully, there won't rise any node-specific windows errors like that edge case.